cypher-shell/src/main/java/org/neo4j/shell/state/BoltStateHandler.java

```diff
@@ -142,9 +140,16 @@ public String getServerVersion() {
             transactionStatements.add(new Statement(cypher, queryParams));
             return Optional.empty();
         } else {
-            List<Statement> transactionStatements = asList(new Statement(cypher, queryParams));
-            BoltResult boltResult = captureResults(transactionStatements).get().get(0);
-            return Optional.of(boltResult);
+            // Note that PERIODIC COMMIT can't execute in a transaction, so if the user has not typed BEGIN, then
+            // the statement should NOT be executed in a transaction.
+            StatementResult statementResult = session.run(new Statement(cypher, queryParams));
```

Comment:
Obviously, leader switches aren't supported outside of transactions with this switch.